config/software/server-jre.rb

```diff
@@ -59,22 +82,27 @@
   # https://www.oracle.com/webfolder/s/digest/7u80checksum.html
   source url:     "http://download.oracle.com/otn-pub/java/jdk/7u80-b15/server-jre-7u80-linux-x64.tar.gz",
          md5:     "6152f8a7561acf795ca4701daa10a965",
-         cookie:  "gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie",
-         warning: "By including the JRE, you accept the terms of the Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX, which can be found at http://www.oracle.com/technetwork/java/javase/terms/license/index.html",
+         cookie:  license_cookie,
+         warning: license_warning,
          unsafe:  true
   relative_path "jdk1.7.0_80"
 end
 
 version "7u25" do
   source url:     "http://download.oracle.com/otn-pub/java/jdk/7u25-b15/server-jre-7u25-linux-x64.tar.gz",
          md5:     "7164bd8619d731a2e8c01d0c60110f80",
-         cookie:  "gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie",
-         warning: "By including the JRE, you accept the terms of the Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX, which can be found at http://www.oracle.com/technetwork/java/javase/terms/license/index.html",
+         cookie:  license_cookie,
+         warning: license_warning,
          unsafe:  true
   relative_path "jdk1.7.0_25"
 end
 
 build do
   mkdir "#{install_dir}/embedded/jre"
-  sync  "#{project_dir}/", "#{install_dir}/embedded/jre"
+  if armhf?
```

Comment:
@elthariel I'm ok with this approach - provided we stick a huge comment here explaining the why on this approach here. Also congrats on the successful port!